tests/imports.pyi

```diff
@@ -3,6 +3,9 @@
 # Note: DO NOT RUN ISORT ON THIS FILE.
 # It's excluded in our pyproject.toml.
 
+# BAD IMPORTS (Y044)
+from __future__ import annotations  # Y044 "from __future__ import annotations" has no effect in stub files.
```

Comment:
Can also move it to the bad imports. I put it up there as I think `__future__` imports have to be the very first import.

Reply:
Having it here is fine I think.